sandbox/apparmor/apparmor_test.go

```diff
@@ -41,6 +44,46 @@ type apparmorSuite struct{}
 
 var _ = Suite(&apparmorSuite{})
 
+func (*apparmorSuite) TestAppArmorFindAppArmorParser(c *C) {
+	mockParserCmd := testutil.MockCommand(c, "apparmor_parser", "")
+	defer mockParserCmd.Restore()
+	restore := apparmor.MockParserSearchPath(mockParserCmd.BinDir())
+	defer restore()
+	restore = apparmor.MockSnapdAppArmorSupportsReexec(func() bool { return false })
+	defer restore()
+	path, args, internal, err := apparmor.FindAppArmorParser()
+	c.Check(path, Equals, mockParserCmd.Exe())
+	c.Check(args, DeepEquals, make([]string, 0))
```

Comment:
I think this should work as well:
```suggestion
	c.Check(args, DeepEquals, []string{})
```